### PR TITLE
Network admin

### PR DESCRIPTION
**Pull request by robbiecarlton**

### CODE REVIEW

**Comment by robbiecarlton** [comment]
@levity can I merge?

**Comment by robbiecarlton** [comment]
@levity can I merge this, and the matching PR in hylo-node?

File: src/containers/network/NetworkAbout.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import React from 'react'
```
Comment (levity) [inline src/containers/network/NetworkAbout.js]: i'm allergic to weird-sounding names -- can this be AboutNetwork instead?

**Comment by levity** [comment]
looks good so far -- feel free to merge this once its counterpart in hylo-node is polished up and merged

**Pull request merged**